.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Feature request for Ask Astro
+about: Suggest an idea for this project
+title: ''
+labels: feature
+assignees: ''
+
+---
+
+**Please describe the feature you'd like to see**
+A clear and concise description of what problem feature solves or what use-case it would enable.
+e.g. "I really wish I could ...", "I'm sad that I can't..."
```

Comment:
```suggestion
<!--
A clear and concise description of what problem feature solves or what use-case it would enable.
e.g. "I really wish I could ...", "I'm sad that I can't..."
-->
```

Reply:
https://github.com/astronomer/ask-astro/pull/92/commits/e4d25af3a21765a5c091840f22f2f8e580f814da